### PR TITLE
Problem: Public Contracts undefined

### PR DESCRIPTION
**Pull request by ttmc**
Solution: Define "Public Contracts"

### CODE REVIEW

**Comment by ttmc** [comment]
Resolves issue #10

File: 1/README.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
 
 ### 2.6. Evolution of Public Contracts
 
+Definition: **Public Conracts** _noun_ - All public APIs and protocols. In other words, read the BEPs (and the things they reference) and you'll have a good start. In particular, see BEP-7 - Definition of the BigchainDB Public API.
```
Comment (vrde) [inline 1/README.md]: Great! Should it belong to `2.1`? All definitions are there. (PS: typo "Conracts").

Comment (ttmc) [reply 1/README.md]: Oh interesting, I didn't notice that some other things were defined there (like "Contributor" and "Maintainer"). Anyway, I think it's fine to put this definition here, right when it's needed, in context.

Comment (ttmc) [reply 1/README.md]: Oops Contracts!

**Pull request merged**